### DISCUSSION TITLE
HDDM "load" function equivalent in HSSM?

### DISCUSSION THREAD

**Discussion by nancyebner** [lnccbrown/HSSM#262] [Q&A] [answered]
I would like to use my own dataset instead of simulating one, in the HDDM package the load function could be used to call a csv file containing my own dataset. I am struggling to find an equivalent in the HSSM framework. It is likely that I am misunderstanding something but I am unable to go on without this step so any help is much appreciated! (ps: I also tried to follow this example (in GoogleColab) using the package-supplied dataset: 

"Load a package-supplied dataset"
cav_data = hssm.load_data('cavanagh_theta')

(https://lnccbrown.github.io/HSSM/#installation) but this is once again a package-supplied dataset, and I would like to use my own. Is this a possibility?

**Comment by digicosmos86** [comment]
Hi,

HSSM just accepts pandas DataFrames now, so you can just use pd.read_csv to get a DataFrame and pass it to HSSM. In a future version we might just do this in HSSM so people will just need to pass a string of the path to the csv.

**Reply by nancyebner** [reply]
Hi, 

Thank you for your answer. I was able to get the dataframe but how do I pass it to HSSM now?
Thanks again.

**Comment by frankmj** [comment, marked as answer]
HI Paul,

once it is a dataframe you can pass it to HSSM in the same way as in the
tutorial using simulated data with whatever model specification you want.
e.g. for the most basic model:

simple_ddm_model = hssm.HSSM(data=dataset)


replacing 'dataset' with the name of  your dataframe, which has to have
columns for response and rt in seconds.

M
